### PR TITLE
Use Java 11 for Jenkins master

### PR DESCRIPTION
**Pull request by mmoll**

### CODE REVIEW

**Comment by mmoll** [comment]
I just tested Java 11 manually and the master starts up fine and while I didn't run any jobs, no messages indicating incompatibilites could be found, so I'm quite confident this works.

The Puppet code is untested.

File: puppet/modules/jenkins_master/manifests/init.pp [hunk 1]
```diff
@@ -191,6 +192,9 @@
     'zentimestamp'                       => {},
   }
 
+  package { $old_packages:
+    ensure => 'absent',
+  } ->
```
Comment (ekohl) [inline puppet/modules/jenkins_master/manifests/init.pp]: This doesn't work. I've added a test to verify this. I'd also expect that you first install the new packages and then remove the old one so there's still a virtual that provides the JRE for Jenkins.

**Comment by mmoll** [comment]
:green_heart: now

**Comment by ekohl** [comment]
Feel free to merge and roll out when you have time.

**Pull request merged**